docs/tool-description.md

```diff
@@ -0,0 +1 @@
+Semgrep accelerates your security journey by swiftly scanning code for known issues and software vulnerabilities with unparalleled efficiency. [Learn more](https://github.com/semgrep/semgrep)
```

Comment:
@claudiacodacy @nicklem can you have a look at this one?

this is the string that will be displayed in Code Patterns page

Reply:
looks good to me 👍

Reply:
It reads more commercial than other tool descriptions, but it's fitting, given the circumstances. LGTM 👍